sys/include/fmt.h

```diff
@@ -395,6 +395,8 @@ size_t fmt_str(char *out, const char *str);
  *
```

Comment:
```suggestion
 * If @p out is NULL, will only return the number of characters that would have
 * been written.
```

while we are at it :)